### PR TITLE
IGNITE-19685 Treat Scalecube's LEAVING event as 'node left' in the physical topology

### PR DESCRIPTION
**Pull request by rpuch**
https://issues.apache.org/jira/browse/IGNITE-19685

### CODE REVIEW

File: modules/network/src/test/java/org/apache/ignite/network/scalecube/ScaleCubeTopologyServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+class ScaleCubeTopologyServiceTest {
+    private final ScaleCubeTopologyService topologyService = new ScaleCubeTopologyService();
+
+    private final Member member1 = new Member("id1", "first", Address.create("host", 1001), "default");
+    private final Member member2 = new Member("id2", "second", Address.create("host", 1002), "default");
+
+    @Test
+    void addedEventAddsNodeToTopology() {
+        topologyService.onMembershipEvent(MembershipEvent.createAdded(member1, null, 1));
+        topologyService.onMembershipEvent(MembershipEvent.createAdded(member2, null, 2));
```
Comment (tkalkirill) [inline modules/network/src/test/java/org/apache/ignite/network/scalecube/ScaleCubeTopologyServiceTest.java:39]:
```suggestion
        addTwoMembers();
```

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/Cluster.java [hunk 2]
```diff
@@ -135,12 +135,28 @@ public void startAndInit(int nodeCount) {
...
+     * Starts the cluster with the given number of nodes and initializes it.
+     *
+     * @param nodeCount Number of nodes in the cluster.
+     * @param nodeBootstrapConfigTemplate Node bootstrap config template to be used for each node started
+     *     with this call.
+     * @param initParametersConfigurator Configure {@link InitParameters} before initializing the cluster.
+     */
+    public void startAndInit(
+            int nodeCount,
+            String nodeBootstrapConfigTemplate,
```
Comment (tkalkirill) [inline modules/runner/src/integrationTest/java/org/apache/ignite/internal/Cluster.java:151]: I propose to move this argument below.

Comment (rpuch) [reply modules/runner/src/integrationTest/java/org/apache/ignite/internal/Cluster.java:151]: Why? Usually it is more convenient to have 'closure-like' arguments last in the arg list

Comment (tkalkirill) [reply modules/runner/src/integrationTest/java/org/apache/ignite/internal/Cluster.java:151]: It just looks better to me when two (or more) methods/constructors have the same order of arguments.

Up to you.

**Review by tkalkirill — APPROVED** [review]

**Pull request merged**